### PR TITLE
Bug 1990493: [e2e][automation] test for storageProfile settings

### PR DESCRIPTION
**Pull request by avivtur**
**Fixes**:
https://bugzilla.redhat.com/show_bug.cgi?id=1990493

polarion (by Gouhua):
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=[CNV-6920](https://issues.redhat.com/browse/CNV-6920)
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=[CNV-6921](https://issues.redhat.com/browse/CNV-6921)
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=[CNV-6922](https://issues.redhat.com/browse/CNV-6922)
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=[CNV-6923](https://issues.redhat.com/browse/CNV-6923)

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/disks/validate-storage-profile.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    cy.deleteResource({
+      kind: 'Namespace',
+      metadata: {
+        name: testName,
+      },
+    });
+  });
+
+  it('ID(CNV-6922) Verify storageProfile in add disk modal', () => {
+    cy.byLegacyTestID('horizontal-link-Disks').click();
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/disks/validate-storage-profile.ts:52]: AFAIU `byTestID` is `data-test=` and `byLegacyTestID` is `data-test-id=`

can you use this PR to refactor all the legacy test ids used in this test to be `data-test=`
(and check if they are used in other tests and refactor there also) ?
it's not fun to see `byTestID` and `byLegacyTestID` in the same test

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/disks/validate-storage-profile.ts:52]: we will do it in a separate PR

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts [hunk 2]
```diff
@@ -29,6 +29,11 @@ declare global {
       uploadFromCLI(dvName: string, ns: string, imagePath: string, size: string): void;
       selectProject(project: string): void;
       createNAD(namespace: string): void;
+      createStorageClass(): void;
```
[the remark this answers is not in the document]
Comment (avivtur) [reply frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts]: moved all functions to same file

**Pull request merged**